arm-network/2016-12-01/swagger/virtualNetworkGateway.json

```diff
@@ -1600,6 +1611,130 @@
       ],	  
       "description": "Response for GetConnectionSharedKey API service call"
     },
+    "IpsecPolicy": {
+      "properties": {
+        "sALifeTimeSeconds": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The Security Association lifetime in seconds for a site to site VPN tunnel."
+        },
+        "sADataSizeKilobytes": {
```

Comment:
> If that is the case, then the entire network api is named incorrectly.

Just synced with the team. If this is true, this is a HUGE ISSUE. Note that we have tons of things depending on those Swagger definitions, and you are basically saying they are wrong. Swaggers *must* be accurate all the way down. What have you tested for? Someone may write their own client to your service, look at the Swagger, go `JSON.parse(rawBody)["someProp"]` but whooops, too bad, your service returned `{ "SomeProp": 42 }`.